pkgs/stdenv/generic/check-meta.nix

```diff
@@ -179,7 +266,7 @@ let
       Known issues:
     '' + (lib.concatStrings (map (issue: " - ${issue}\n") attrs.meta.knownVulnerabilities)) + ''
 
-        You can install it anyway by allowing this package, using the
+        You can install it anyways by allowing this package, using the
```

Comment:
?

Reply:
"anyway" is the more conventional phrasing.